### ISSUE TITLE
[v6 iOS] waitForRender on SetRoot doesn't work with a SideMenu 

### ISSUE THREAD

**Issue by RomualdPercereau** [wix/react-native-navigation#6000]
### Issue Description

Using the waitForRender option with a SideMenu doesn't work. 

### This will blink : 
```js
  Navigation.setDefaultOptions({
    animations: {
      setRoot: {
        waitForRender: true
      }
    }
  })
  Navigation.setRoot({
    root: {
      sideMenu: {
        left: {
          component: {
            name: "av.MenuContainer"
          }
        },
        center: {
          stack: {
            id: "k",
            children: [
              {
                component: {
                  name: "av.NavBarContainer",
                  passProps: {}
                }
              }
            ]
          }
        }
      }
    }
  })

```

### This will NOT blink : 
```js
  Navigation.setDefaultOptions({
    animations: {
      setRoot: {
        waitForRender: true
      }
    }
  })
  Navigation.setRoot({
    root: {
      stack: {
        id: "k",
        children: [
          {
            component: {
              name: "av.NavBarContainer",
              passProps: {}
            }
          }
        ]
      }
    }
  })

```
![ezgif-7-60cda80e32c3](https://user-images.githubusercontent.com/4194506/75964936-5aea5c00-5ec8-11ea-9bfb-2665da4122b8.gif)
---
### Environment
* React Native Navigation version: FILL 6.0.1
* React Native version: 0.61.5
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Tested on iOS 9, iOS13 Debug and release

**Comment by didrikkvanvik** [comment]
I am experiencing the same issue with `bottomtabs` as well. Any progress on this issue?

**Comment by danilobuerger** [comment]
@didrikkvanvik bottomtabs should be fixed by #6689

**Comment by danilobuerger** [comment]
Should be fixed in 7.2.0
